### PR TITLE
Presedens related changes in the documentation

### PR DESCRIPTION
**Pull request by vulkodava**
Document the new PrecedentRegistry Permission endpoints
Document the new Presedens object
Document the new Presedens related List Values (presedensStatus,
    presedensHjemmel and rettskildefaktor)

### CODE REVIEW

File: version1/02 - Object types.md [hunk 1]
```diff
@@ -295,6 +295,7 @@ The following notation is used in the tables below:
...
 |           | ref       | refKryssreferanseFraRegistrering |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that cross reference the current **AbstraktMappe**                                                                                                                                                                                                                |
+|           | ref       | refPresedens                     |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **Presedens** that cross reference the current **Saksmappe**                                                                                                                                                                                                                               |
```
Comment (teodoradobreva) [inline version1/02 - Object types.md]: I think that the `that cross reference the current **Saksmappe**` part should refer to cross references only, and should be removed from here. Same for Journalpost.

File: version1/03 - Code lists.md [hunk 2]
```diff
@@ -226,3 +226,27 @@ No default values are present in this list and must be configured in the system
...
+| F    | Foreldet              |  &nbsp; |
+
+No default values are present in this list and must be configured in the system before use.
+
+### presedensHjemmel (M311)
+
+| Code | Name | Comment |
+|:-----|:-----|:--------|
+|      |      |  &nbsp; |
+No default values are present in this list and must be configured in the system before use.
```
Comment (teodoradobreva) [inline version1/03 - Code lists.md:243]: An empty line is needed before this one for better formatting.

File: version1/03 - Code lists.md [hunk 3]
```diff
@@ -226,3 +226,27 @@ No default values are present in this list and must be configured in the system
 |:-----|:----------------------|:--------|
 | IMER | Intern merknad        |         |
 | MER  | Merknad               |  &nbsp; |
+
+No default values are present in this list and must be configured in the system before use.
```
Comment (teodoradobreva) [inline version1/03 - Code lists.md]: Should be removed.

File: version1/03 - Code lists.md [hunk 4]
```diff
@@ -226,3 +226,27 @@ No default values are present in this list and must be configured in the system
...
+
+### presedensStatus (M056)
+
+| Code | Name                  | Comment |
+|:-----|:----------------------|:--------|
+| G    | Gjeldende             |         |
+| F    | Foreldet              |  &nbsp; |
+
+No default values are present in this list and must be configured in the system before use.
```
Comment (teodoradobreva) [inline version1/03 - Code lists.md]: Should be removed.

File: version1/03 - Code lists.md [hunk 5]
```diff
@@ -226,3 +226,27 @@ No default values are present in this list and must be configured in the system
...
+### presedensHjemmel (M311)
+
+| Code | Name | Comment |
+|:-----|:-----|:--------|
+|      |      |  &nbsp; |
+No default values are present in this list and must be configured in the system before use.
+
+### rettskildefaktor (M312)
+
+| Code | Name | Comment |
```
Comment (teodoradobreva) [inline version1/03 - Code lists.md:247]: You can add `No default values are present in this list and must be configured in the system before use.` after this table.

File: version1/02 - Object types.md [hunk 6]
```diff
@@ -757,6 +759,30 @@ The following notation is used in the tables below:
...
+| M628      | timestamp | presedensGodkjentDato  |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M629      | string    | presedensGodkjentAv    |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+|           |           |                        |          |             |                  |  |      |      |       |      |      |        |                                                                                                                                |
+| M056      | string    | presedensStatus        |          |      x      | presedensStatus  |  |  x   |  x   |   x   |      |      |        |                                                                                                                                |
```
Comment (teodoradobreva) [inline version1/02 - Object types.md]: Should have a mark for not null.

File: version1/02 - Object types.md [hunk 7]
```diff
@@ -757,6 +759,30 @@ The following notation is used in the tables below:
...
+| M311      | string    | presedensHjemmel       |          |             | presedensHjemmel |  |  x   |  x   |   x   |      |      |        |                                                                                                                                |
+| M312      | string    | rettskildefaktor       |          |             | rettskildefaktor |  |  x   |  x   |   x   |      |      |        |                                                                                                                                |
```
Comment (teodoradobreva) [inline version1/02 - Object types.md]: Should have a mark for not null.

File: version1/02 - Object types.md [hunk 8]
```diff
@@ -757,6 +759,30 @@ The following notation is used in the tables below:
...
+| M021      | string    | beskrivelse            |          |             |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M111      | date      | presedensDato          |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M628      | timestamp | presedensGodkjentDato  |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M629      | string    | presedensGodkjentAv    |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
```
Comment (teodoradobreva) [inline version1/02 - Object types.md]: `presedensGodkjentAvIdent` is missing (if naming stays like this).

Comment (teodoradobreva) [inline version1/02 - Object types.md]: presedensGodkjentAv and its corresponding ID field should be cw on save, as you cannot save one without saving the other.

**Review by teodoradobreva — CHANGES_REQUESTED** [review]

File: version1/02 - Object types.md (hunk 8 above)
Comment (teodoradobreva) [reply version1/02 - Object types.md]: Actually, seems like we would probably name the field presedensGodkjentAvBrukerIdent.

File: version1/02 - Object types.md (hunk 1 above)
Comment (vulkodava) [reply version1/02 - Object types.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/03 - Code lists.md (hunk 2 above)
Comment (vulkodava) [reply version1/03 - Code lists.md:243]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/03 - Code lists.md (hunk 3 above)
Comment (vulkodava) [reply version1/03 - Code lists.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/03 - Code lists.md (hunk 4 above)
Comment (vulkodava) [reply version1/03 - Code lists.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/03 - Code lists.md (hunk 5 above)
Comment (vulkodava) [reply version1/03 - Code lists.md:247]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/02 - Object types.md (hunk 6 above)
Comment (vulkodava) [reply version1/02 - Object types.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/02 - Object types.md (hunk 7 above)
Comment (vulkodava) [reply version1/02 - Object types.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

File: version1/02 - Object types.md (hunk 8 above)
Comment (vulkodava) [reply version1/02 - Object types.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

Comment (vulkodava) [reply version1/02 - Object types.md]: Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6

**Review by teodoradobreva — APPROVED** [review]

**Review by ivaylomitrev — APPROVED** [review]

**Pull request merged**